packages/react/src/themes/teams-dark/components/Menu/menuVariables.ts

```diff
@@ -7,6 +7,8 @@ export default (siteVars: any): Partial<MenuVariables> => ({
   focusedBorder: `solid ${pxToRem(1)} ${siteVars.colors.white}`,
   focusedBackgroundColor: 'transparent',
 
+  primaryActiveBorderColor: siteVars.brand06,
```

Comment:
I would consider renaming this to something without "primary" in it. It might make people think they need the `primary` prop, which they don't.

Reply:
Correct. Renamed it.